test/module.mk

```diff
@@ -830,6 +845,31 @@ $(DIR)/test_loopfunctions.x: $(LIBCMSSM)
 
 $(DIR)/test_sfermions.x: $(LIBCMSSM)
 
+$(DIR)/test_looplibrary_softsusy.cpp: $(DIR)/test_looplibrary.cpp.in
+	@sed 's/looplibrary_/looplibrary_softsusy/' \
+		 test/test_looplibrary.cpp.in > test/test_looplibrary_softsusy.cpp
+
+ifeq ($(ENABLE_COLLIER), yes)
+$(DIR)/test_looplibrary_collier.cpp: $(DIR)/test_looplibrary.cpp.in
+	@sed -e 's/LIBRARY_TYPE 0/LIBRARY_TYPE 1/' \
```

Comment:
Please avoid the `sed` command here and use a simple `mv` command to create the files. Please move the logic into the `test_looplibrary.cpp.in` file, using for example a preprocessor definition:
~~~
-DLIBRARY_TYPE=1
~~~
etc. Inside `test_looplibrary.cpp.in` you may then use `#if` statements to include the files and define the library name.

Reason: Our build system is already very complex and I'd like to replace it in the future by something more robust. Using `sed` replacements here makes this transition harder and makes our build system even more complicated. The fewer scripting we use here, the better.

Reply:
My comment also applies to the `sed` replacement rules in the following lines.